label_studio/webhooks/api.py

```diff
@@ -0,0 +1,87 @@
+import logging
+
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import generics
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+from rest_framework.permissions import IsAuthenticated, AllowAny
+
+from .models import Webhook, WebhookAction
+from .serializers import WebhookSerializer
+
+
+class WebhookListAPI(generics.ListCreateAPIView):
+    queryset = Webhook.objects.all()
+    serializer_class = WebhookSerializer
+    permission_classes = [IsAuthenticated]
+
+    def get_queryset(self):
+        return Webhook.objects.filter(organization=self.request.user.active_organization)
+
+    def perform_create(self, serializer):
+        serializer.save(organization=self.request.user.active_organization)
+
+    @swagger_auto_schema(
```

Comment:
We are trying to start writing swagger_auto_schema before a class implementation, like here: 

![image](https://user-images.githubusercontent.com/501780/122137791-20eb6200-ce4e-11eb-8647-169da80e2031.png)

or 

![image](https://user-images.githubusercontent.com/501780/122137903-57c17800-ce4e-11eb-82f1-d073c8eabe4b.png)

in these cases you don't need to implement dummy def post(), def get(), ... and all descriptions are collected in one place.